ipld/cid/src/json.rs

```diff
@@ -41,3 +41,27 @@ struct CidMap {
     #[serde(rename = "/")]
     cid: String,
 }
+
+pub mod vec {
+    use super::*;
+    use forest_json_utils::GoVecVisitor;
+    use serde::ser::SerializeSeq;
+
+    pub fn serialize<S>(m: &[Cid], serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: Serializer,
+    {
+        let mut seq = serializer.serialize_seq(Some(m.len()))?;
+        for e in m {
+            seq.serialize_element(&CidJsonRef(e))?;
+        }
+        seq.end()
+    }
+
+    pub fn deserialize<'de, D>(deserializer: D) -> Result<Vec<Cid>, D::Error>
+    where
+        D: Deserializer<'de>,
+    {
+        deserializer.deserialize_any(GoVecVisitor::<Cid, CidJson>::new())
+    }
+}
```

Comment:
Since this is duplicated with the tipset keys, can you just switch that implementation to call this to not have redundancy?